src/microbe_stage/gui/PatchDetailsPanel.tscn

```diff
@@ -302,6 +303,25 @@ layout_mode = 2
 text = "CO₂"
 label_settings = ExtResource("15_ng6rp")
 
+[node name="Other" type="HBoxContainer" parent="NormalContent/MarginContainer/ScrollContainer/VBoxContainer/PatchDetailsContainer/AtmosphericGasses"]
+layout_mode = 2
+size_flags_horizontal = 3
+mouse_filter = 2
+
+[node name="TextureRect" type="TextureRect" parent="NormalContent/MarginContainer/ScrollContainer/VBoxContainer/PatchDetailsContainer/AtmosphericGasses/Other"]
+custom_minimum_size = Vector2(30, 25)
+layout_mode = 2
+size_flags_vertical = 4
+tooltip_text = "CARBON_DIOXIDE"
+texture = ExtResource("16_baud4")
+expand_mode = 1
+
+[node name="Label" type="Label" parent="NormalContent/MarginContainer/ScrollContainer/VBoxContainer/PatchDetailsContainer/AtmosphericGasses/Other"]
+editor_description = "PLACEHOLDER"
+layout_mode = 2
+text = "Other"
```

Comment:
Shouldn't the text be translatable as this will be displayed like it is.?

Reply:
I was wondering too, but other compounds are written lowercase too...

Reply:
Probably doesn't matter as it gets replaced with numbers

Reply:
Well that's maybe a slight problem because if these don't have any text, then the player is going to wonder what the mystery compound is that takes up a ton of space.